content/modules/redistimeseries/release-notes/redistimeseries-1.6-release-notes.md

```diff
@@ -0,0 +1,36 @@
+---
+Title: RedisTimeSeries 1.6 release notes
+linkTitle: v1.6 (January 2022)
+description: RedisTimeSeries 1.6 adds support for aggregating across multiple time series (multi-key). Before this version, queries such as “the maximum observed value of a set of time series” needed to be calculated client-side. Such queries can now be computed server-side, leveraging the heart of RedisGears ([LibMR](https://github.com/RedisGears/LibMR)) for clustered databases.
```

Comment:
```suggestion
description: Added support for aggregating across multiple time series (multi-key). Can compute queries such as “the maximum observed value of a set of time series” server-side instead of client-side.
```